hazelcast/src/main/java/com/hazelcast/cache/impl/operation/CacheSetExpiryPolicyBackupOperation.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.cache.impl.operation;
+
+import com.hazelcast.cache.impl.CacheDataSerializerHook;
+import com.hazelcast.cache.impl.CacheEntryViews;
+import com.hazelcast.cache.impl.record.CacheRecord;
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.nio.serialization.Data;
+import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
+import com.hazelcast.spi.BackupOperation;
+import com.hazelcast.spi.ServiceNamespaceAware;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CacheSetExpiryPolicyBackupOperation
+        extends CacheOperation
+        implements BackupOperation, IdentifiedDataSerializable, ServiceNamespaceAware {
```

Comment:
only implementing `BackupOperation` should be sufficient here, others (`IdentifiedDataSerializable and ServiceNamespaceAware`) are already exist in `CacheOperation`